### ISSUE TITLE
Model errors and Aristocrat not receiving due passes

### ISSUE THREAD

**Issue by GBEYOND0613** [itsamewayu/OLFallenDoll#1181]
**Describe the bug**

切换为 Juzi 时， 模型出现错误

未收到应有的通行证或对应的新英格兰磅

**To Reproduce**
1. Go to '后宫模式'
2. Click on '角色'
3. 切换为 Juzi

**Expected behavior**

**Screenshots**
角色模型错误：
<img width="1280" alt="image" src="https://github.com/itsamewayu/OLFallenDoll/assets/57569566/157b93d5-d235-4402-b8b4-120b8aefaead">

未收到应有的奖励：
![image](https://github.com/itsamewayu/OLFallenDoll/assets/57569566/50b65c57-7d9a-4bca-b057-061a37f24c8c)

**Desktop (please complete the following information):**
 - OS: W11
 - Game Version: 0.8.1
 Complete the following Computer Hardware Configurations if necessary:
 - Processor(CPU): i7-13700HX
 - Graphics Card: NVIDIA GeForce RTX 4080 Laptop GPU
 - Memory: 32GB

**Comment by GBEYOND0613** [comment]
No, only receive the patreon NEP for membership, not receive the character's NEP

PS. I change the Membership to "passerby" to receive the update because the "Aristocrat" is too expensive and it said I will continue receiving every updates and monthly bundles till end of development even after you no longer make a pledge

**Comment by Awfulwaffle1** [comment]
@GBEYOND0613 Please create a support ticket through discord so I we can have the devs take a look.
